tools/releng/ls-missing-release-builds

```diff
@@ -51,7 +51,7 @@ function usage() {
 }
 
 # Specify the type of platform build to do.
-PLATFORM_BUILD_FLAVOR='smartos-and-triton'
+PLATFORM_BUILD_FLAVOR='triton-and-smartos'
```

Comment:
Ha ha. I see that sdcrelease.html doc had it the other way.

Reply:
yeah, and the correct one is what's there for https://jenkins.joyent.us/job/platform/configure and https://github.com/joyent/smartos-live/blob/master/tools/build_jenkins#L164